README.md

```diff
@@ -74,18 +74,25 @@ Hint: To create S3 buckets from the command line, use `aws s3 mb --region REGION
     download the service account key JSON file.
 
 3.  In `environment.local`, set the environment variable `GOOGLE_APPLICATION_CREDENTIALS` to the path of the service
-    account key JSON file.
+    account key JSON file. Alternatively, keep the default for the environment variable and rename the file accordingly.
```

Comment:
`GOOGLE_APPLICATION_CREDENTIALS` has been removed from `environment`.

The command example should perhaps read:
`cat /path/to/gcp-credentials.json | scripts/set_secret.py --secret-name $GOOGLE_APPLICATION_CREDENTIALS_SECRETS_NAME`